testdata/p4_16_samples/convenient_init.p4

```diff
@@ -0,0 +1,47 @@
+#include <core.p4>
+
+enum bit<8> myEnum { One = 1, Two = 2, Three = 3 }
+
+enum myEnum1 {e1, e2, e3}
+
+
+header H {
+    int<8> i;
+    bool   b;
+    bit<14> l;
+    bit<1> b1;
+}
+
+
+struct S {
+    myEnum      val;
+    myEnum1     val1;
+    error       errorNoOne;
+    H           h;
+    bit<1>              z;
+    bool        b;
+    int<8>      i;
+}
+
+
+extern void f<D>(in D data);
+control c(inout bit<1> r) {
+    S s_0;
+    H h_0;
+    bit<1> tmp;
+    apply {
+        S s_1;
+        s_1 = { b = false, h={3, true, ...}, ...};
```

Comment:
I would like to please request that this test program be changed slightly to make it easier for a person to check the program against the frontend and midend versions, for example by making the following changes in variable names:

+ s_0 -> sa
+ s_1 -> sb
+ h_0 -> ha
+ h -> hb

With the suffixes like `_0` and `_1`, the automatic renaming that some compiler passes do can make it very difficult to see how the renamed variables correspond with the original ones.